app/src/renderer/vuex/modules/send.js

```diff
@@ -153,8 +153,8 @@ function assertOk(res) {
     return res.forEach(assertOk)
   }
 
-  if (res.check_tx.code || res.deliver_tx.code) {
-    const message = res.check_tx.log || res.deliver_tx.log
+  if (!(res.height && res.txhash)) {
```

Comment:
Why both?

Reply:
safety, in theory txhash should be enough